### PR TITLE
tiup: add 3 docs disable/display/list

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
add 3 TiUP docs:

- tiup/tiup-component-cluster-disable.md
- tiup/tiup-component-cluster-display.md
- tiup/tiup-component-cluster-list.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4868
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-component-cluster-display.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+If you want to see the operation status of each component in the cluster, it is obviously inefficient to log in to each machine one by one. Therefore, tiup-cluster provides the `tiup cluster display` command to efficiently complete this task.
+
+## Syntax
+
+```shell
+tiup cluster display <cluster-name> [flags]
+```
+
+`<cluster-name>`: the name of the cluster to operate. If you forget the cluster name, you can check it by the [cluster list](/tiup/tiup-component-cluster-list.md).
```
Comment (ran-huang) [inline tiup/tiup-component-cluster-display.md:15]:
```suggestion
`<cluster-name>`: the name of the cluster to operate on. If you forget the cluster name, you can check it in the [cluster list](/tiup/tiup-component-cluster-list.md).
```

File: tiup/tiup-component-cluster-display.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+- By default, all node information of the entire cluster is displayed. After adding the `--dashboard` option, only dashboard information is displayed.
+- Data type: `BOOLEAN`
+- Default: false
+
+### -N, --node
+
+- Data type: `strings`
+- Default: `[]`, which means all nodes.
+- Specifies the node to query. If not specified, it means all nodes. The value of this option is a comma-separated list of node IDs. The node ID is the first column of the [cluster status](/tiup/tiup-component-cluster-display.md) table.
```
Comment (ran-huang) [inline tiup/tiup-component-cluster-display.md:29]:
```suggestion
- Specifies the node to query. If not specified, all nodes are queried. The value of this option is a comma-separated list of node IDs. The node ID is the first column of the [cluster status](/tiup/tiup-component-cluster-display.md) table.
```

File: tiup/tiup-component-cluster-display.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+> **Note:**
+>
+> If `-R, --role` are specified at the same time, then the service status in their intersection is queried.
```
Comment (ran-huang) [inline tiup/tiup-component-cluster-display.md:33]: same as previous comments

File: tiup/tiup-component-cluster-display.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+### -R, --role strings
+
+- Data type: `strings`
+- Default: `[]`, which means all roles.
+- Specifies the role to query. If not specified, it means all roles. The value of this option is a comma-separated list of node roles. The role is the second column of the [cluster status](/tiup/tiup-component-cluster-display.md) table.
```
Comment (ran-huang) [inline tiup/tiup-component-cluster-display.md:39]:
```suggestion
- Specifies the role to query. If not specified, all roles are queried. The value of this option is a comma-separated list of node roles. The role is the second column of the [cluster status](/tiup/tiup-component-cluster-display.md) table.
```

File: tiup/tiup-component-cluster-display.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+> **Note:**
+>
+> If `-N, --node` are specified at the same time, then the service status in their intersection is queried.
```
Comment (ran-huang) [inline tiup/tiup-component-cluster-display.md:43]: ditto

File: tiup/tiup-component-cluster-list.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+---
+title: tiup cluster list
+---
+
+# tiup cluster list
+
+tiup-cluster supports the deployment of multiple clusters using the same central control machine. You can use the `tiup cluster list` command to view which clusters are deployed by the currently logged-in user using this central control machine.
```
Comment (ran-huang) [inline tiup/tiup-component-cluster-list.md:7]:
```suggestion
tiup-cluster supports deploying multiple clusters using the same control machine. The `tiup cluster list` command outputs all clusters deployed by the currently logged-in user using this control machine.
```
`中控机` is `control machine`. You can check our Shimo term base.

File: tiup/tiup-component-cluster-list.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+> **Note:**
+>
+> The deployed cluster data is placed in the `~/.tiup/storage/cluster/clusters/` directory by default, so on the same central control machine, the currently logged-in user cannot view the clusters deployed by other users.
```
Comment (ran-huang) [inline tiup/tiup-component-cluster-list.md:11]:
```suggestion
> The deployed cluster data is stored in the `~/.tiup/storage/cluster/clusters/` directory by default, so on the same control machine, the currently logged-in user cannot view the clusters deployed by other users.
```

File: tiup/tiup-component-cluster-list.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+- Default: false
+
+## Outputs
+
+Outputs the table with the following fields:
+
+- Name: the cluster name
+- User: the deployment user
+- Version: the cluster version
+- Path: the path of the cluster deployment data on the central control machine
```
Comment (ran-huang) [inline tiup/tiup-component-cluster-list.md:34]:
```suggestion
- Path: the path of the cluster deployment data on the control machine
```

File: tiup/tiup-component-cluster-list.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+- PrivateKey: the path of the private key that is connected to the cluster
```
Comment (ran-huang) [inline tiup/tiup-component-cluster-list.md:35]:
```suggestion
- PrivateKey: the path of the private key that is used to connect the cluster
```

File: tiflash/tiflash-configuration.md [hunk 10]
```diff
@@ -65,7 +65,7 @@ delta_index_cache_size = 0
 
 ## Storage paths settings take effect starting from v4.0.9
 [storage]
-    ## [Experimental] New in v5.0. This item limits the total write rate of background tasks in bytes per second. It is not recommanded to use this experimental feature in a production environment.
+    ## [Experimental] New in v5.0. This item limits the total write rate of background tasks in bytes per second. It is not recommended to use this experimental feature in a production environment.
```
Comment (lucklove) [inline tiflash/tiflash-configuration.md:68]: It seems not related to TiUP?

File: tiup/tiup-component-cluster-display.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
+---
+title: tiup cluster display
+---
+
+# tiup cluster display
+
+If you want to see the operation status of each component in the cluster, it is obviously inefficient to log in to each machine one by one. Therefore, tiup-cluster provides the `tiup cluster display` command to efficiently complete this task.
```
Comment (lucklove) [inline tiup/tiup-component-cluster-display.md:7]: 登录到每台机器应该是翻译成 "login to each machine" 还是 "log in to each machine" 呀？

File: tiup/tiup-component-cluster-display.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+
+## Syntax
+
+```shell
+tiup cluster display <cluster-name> [flags]
+```
+
+`<cluster-name>`: the name of the cluster to operate on. If you forget the cluster name, you can check it in the [cluster list](/tiup/tiup-component-cluster-list.md).
```
Comment (lucklove) [inline tiup/tiup-component-cluster-display.md:15]:
```suggestion
`<cluster-name>`: the name of the cluster to operate on. If you forget the cluster name, you can check it with the command [cluster list](/tiup/tiup-component-cluster-list.md).
```

File: tiup/tiup-component-cluster-display.md [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+
+## Options
+
+### --dashboard
+
+- By default, all node information of the entire cluster is displayed. After adding the `--dashboard` option, only dashboard information is displayed.
```
Comment (lucklove) [inline tiup/tiup-component-cluster-display.md:21]:
```suggestion
- By default, all node information of the entire cluster is displayed. With the `--dashboard` option, only dashboard information is displayed.
```

**Review by ran-huang — APPROVED** [review]

File: tiup/tiup-component-cluster-display.md (hunk 11 above)
Comment (ran-huang) [reply tiup/tiup-component-cluster-display.md:7]: `log in` 吧，动词短语，`login` 是名词

**Comment by Joyinqin** [comment]
@lucklove  PTAL again thx~
